app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java

```diff
@@ -722,9 +761,12 @@ public void removeLike(String userId, String otherUserId) {
                 new String[]{userId, otherUserId});
     }
 
-    // precondition: userId < otherUserId. Since matches should not have duplicate of
-    // (userId, otherUserId) and (otherUserId, userId), we should respect the order when
-    // adding/deleting records of userId pair.
+    /**
+     * Removes a matched pair fom the database.
+     * Precondition: userId < otherUserId.
```

Comment:
It got excluded why we have this precondition. Can we include it back?

Reply:
done